src/components/Editor/Editor.css

```diff
@@ -98,7 +98,7 @@ html[dir="rtl"] .editor-mount {
 }
 
 .highlight-line .CodeMirror-line {
-  animation: fade-highlight-out 1.5s normal forwards;
+  animation: fade-highlight-out 3s normal forwards;
```

Comment:
hmm - i think this also happens when you click on a breakpoint in the breakpoint list or you select a frame in the call stack...

Is that correct?